compiler/next/include/chpl/queries/ErrorMessage.h

```diff
@@ -36,8 +36,17 @@ namespace chpl {
   is saved (in the event it needs to be reported again).
  */
 class ErrorMessage final {
+ public:
+  enum Kind {
+    NOTE,
+    WARNING,
+    SYNTAX,
+    ERROR,
+    FATAL
```

Comment:
I think we should remove FATAL here and use SYNTAX and ERROR. At least until we know of a case where the error is so bad we can't continue running queries with ErroneousExpression / ErroneousType etc.